### PR TITLE
FMK-1104 : Enable power management cpufreq/sys_cpufreq test in kpet-db

### PR DESCRIPTION
**Pull request by idorax**
Case `cpufreq/sys_cpufreq` is re-written because it is using a complicated case parser. Without using that parser, the case looks pretty straightforward and it is also easy to maintain.

A manual test is done on a system having Intel CPU.
* http://pastebin.test.redhat.com/809878

### CODE REVIEW

File: power-management/cpufreq/sys_cpufreq/Makefile [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	@echo "Path:         $(TEST_DIR)" >> $(METADATA)
+	@echo "Description:  Confirm RAPL is supported" >> $(METADATA)
+	@echo "Type:         Sanity" >> $(METADATA)
+	@echo "RunFor:       kernel" >> $(METADATA)
+	@echo "Requires:     kernel" >> $(METADATA)
+	@echo "Requires:     kernel-tools" >> $(METADATA)
+	@echo "Requires:     virt-what" >> $(METADATA)
+	@echo "Requires:     bc" >> $(METADATA)
+	@echo "Requires:     python2-lxml" >> $(METADATA)
+	@echo "Requires:     python3-lxml" >> $(METADATA)
```
[the remark this answers is not in the document]
Comment (2019210654) [reply power-management/cpufreq/sys_cpufreq/Makefile]: @emanchado , python2-lxml/python3-lxml is required by beaker to create report. Some system (e.g. RHEL7) uses python2-lxml, some system (e.g. RHEL8) uses python3-lxml.  Hence I added both of them. `bc` is removed.

[the remark this answers is not in the document]
Comment (2019210654) [reply power-management/cpufreq/sys_cpufreq/Makefile]: Here is a sample log without `python?-lxml`.
```
Python ImportError: No module named 'lxml'
Exiting unsuccessfully.
:: [ 05:37:23 ] :: [ WARNING  ] :: cannot create journal.xml due to missing python lxml module
```
* http://pastebin.test.redhat.com/810858

**Comment by emanchado** [comment]
Excellent! Should I merge?

**Comment by idorax** [comment]
Hi @emanchado, please help to merge it, thanks!

**Pull request merged**